src/tests/functional/shared_test_classes/include/shared_test_classes/subgraph/gather_weights_decompression.hpp

```diff
@@ -25,29 +26,8 @@ namespace test {
  *             Gather
  */
 
-struct GWDShapeParams {
-    GWDShapeParams() = default;
-    GWDShapeParams(ov::Shape data_shape,
-                InputShape indices_shape,
-                int axis,
-                int64_t batch_dims,
-                int decompression_group_size = -1)
-        : data_shape(std::move(data_shape)),
-          indices_shape(std::move(indices_shape)),
-          axis(axis),
-          batch_dims(batch_dims),
-          decompression_group_size(decompression_group_size) {}
-
-    ov::Shape data_shape;
-    InputShape indices_shape;
-    int axis;
-    int64_t batch_dims;
-    // Decompression group size. If the value is equal to -1, ordinary decompression is used
-    int decompression_group_size;
-};
```

Comment:
This code is moved to weights_decompression_builders.hpp, `GWDShapeParams` is renamed to `GatherDecompressionShapeParams`